.github/workflows/admin.yml

```diff
@@ -76,7 +76,7 @@ jobs:
         run: make test-cover
 
       - name: Upload coverage results
-        uses: percona-platform/codecov-action@v3
+        uses: codecov/codecov-action@v3
```

Comment:
@atymchuk please add original codecov action to allow list of actions

Reply:
I checked - it's there, so we should be good to go